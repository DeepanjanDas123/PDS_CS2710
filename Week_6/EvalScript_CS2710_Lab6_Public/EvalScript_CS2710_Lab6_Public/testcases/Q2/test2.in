6
NEW PUSH_END 10 PUSH_END 5 PUSH_END 1 PUSH_BEGIN 99 ROTATE 2