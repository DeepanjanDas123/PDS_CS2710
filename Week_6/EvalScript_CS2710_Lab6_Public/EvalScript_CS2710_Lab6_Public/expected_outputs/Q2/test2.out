5 1 99 10